src/components/forms/DocumentForm.tsx

```diff
@@ -265,7 +267,7 @@ export const DocumentForm = ({
               )
             }}
           />
-          <ButtonGroup>
+          <ButtonGroup isDisabled={canAccess}>
```

Comment:
Is this logic right? I would have thought `isDisabled= !canAccess`

Reply:
Have resolved.